__tests__/logs.test.js

```diff
@@ -0,0 +1,131 @@
+const fs = require('fs');
+const pool = require('../lib/utils/pool');
+const request = require('supertest');
+const app = require('../lib/app');
+const Log = require('../lib/models/log');
+const Recipe = require('../lib/models/recipe');
+
+describe('log routes', () => {
+  let recipe;
+
+  beforeEach(async() => {
+    await pool.query(fs.readFileSync('./sql/setup.sql', 'utf-8'));
+
+    recipe = await Recipe.insert({
+      name: 'cookies',
+      directions: [
+        'preheat oven to 375',
+        'mix ingredients',
+        'put dough on cookie sheet',
+        'bake for 10 minutes'
+      ]
+    });
+  });
+    
+  afterAll(() => {
+    return pool.end();
+  });
+
+  it('creates a log', async() => {
+
+    const log = await Log.insert({
+      dateOfEvent: '12/24/20',
+      notes: 'here are some notes',
+      rating: '10/10',
+      recipeId: recipe.id
+    });
+
+    return request(app)
+      .post('/api/v1/logs')
+      .send(log)
+      .then(res => {
+        expect(res.body).toEqual({ ...log, id: '2' });
+      });
+  });
+
+  it('returns all logs', async() => {
+
+    const logs = await Promise.all([
+      {
+        dateOfEvent: '12/24/20',
+        notes: 'here are some notes',
+        rating: '10/10',
+        recipeId: recipe.id
+      },
+      {
+        dateOfEvent: '12/24/20',
+        notes: 'here are some notes',
+        rating: '10/10',
+        recipeId: recipe.id
+      },
+      {
+        dateOfEvent: '12/24/20',
+        notes: 'here are some notes',
+        rating: '10/10',
+        recipeId: recipe.id
+      }
+    ].map(log => Log.insert(log)));
+
```

Comment:
I feel the logs should be a one to many or many to many with the recipe model. Please change to reflect that.